### DISCUSSION TITLE
Sliders not working anymore.

### DISCUSSION THREAD

**Discussion by djbb84** [artisan-roaster-scope/artisan#933] [General]
So here's the scoop. My sliders aren't working so I cannot input the changes I'm making. 

Here's the background:I am running Mac OS Big Sur on a 2015 MacBook Pro 13in Retina display....

I was using the previous version of Artisan (Sweet Maria's release) and I decided to update to the current release (v2.6.0)

I downloaded and installed it from my desktop and ran it. It was working fine, I roasted a few times, but then after accidentally opening a second window and entering viewer mode (I am pretty new to artisan and didn't know what viewer mode was- I had to reset sliders a few times before on the Sweet Maria's version (b/c I must have been in in viewer mode)

Well I did it again and was in viewer mode, but since then I CANNOT see my sliders anymore. 

What I've done to help fix that: 
With the help of Michael Herbert I deleted artisan (both versions) and reinstalled the current version one. No luck, I restarted my computer and told it not to reopen windows, no luck. I've looked into settings and I do NOT have 
"Open completed roast in viewer" selected...

It's still not working..I need help please. I just can't get my sliders to work. In fact the dialogue is greyed out when I start a new roast (reset and START, so that I can't select them (and they're unchecked).

Please help!! 

-Daniel

**Comment by MAKOMO** [comment]
Restart your computer. You have a hanging Artisan process. If you know how, you can also kill it.

**Reply by djbb84** [reply]
I did a restart, and made sure to uncheck "reopen windows"...it didn't work

**Reply by poundy** [reply]
Daniel, can you show us screen shots of the single Artisan window that opens? 
As Marko mentioned, the common issue here is that Artisan is detecting another Artisan process already running and so runs the second instance in viewer mode. 
I'm no help on the Mac side sorry, so I can't help diagnose / find what the process is. 
Do you have access to another device, mac or PC, that you can take your config over onto and try from there ?
